frontend/app/controllers/Joiner.scala

```diff
@@ -259,14 +259,17 @@ object Joiner extends Controller with ActivityTracking
         // errors due to user's card are logged at WARN level as they are not logic errors
         case error: Stripe.Error =>
           logger.warn(s"Stripe API call returned error: \n\t$error \n\tuser=${request.user.id}")
+          setBehaviourNote(tier.name, error.code)(request)
           Forbidden(Json.toJson(error))
 
         case error: PaymentGatewayError =>
+          setBehaviourNote(tier.name, error.code)(request)
           handlePaymentGatewayError(error, request.user.id, tier.name, idRequest.trackingParameters, formData.deliveryAddress.countryName)
 
         case error =>
           salesforceService.metrics.putFailSignUp(tier)
           logger.error(s"User ${request.user.id} could not become ${tier.name} member: ${idRequest.trackingParameters}", error)
+          setBehaviourNote(tier.name, "card_error")(request)
```

Comment:
You're right, but I don't know exactly what'll come out of here yet. The only intentional errors I've been able to create using the Stripe test card numbers generate a typical stack trace error message. I considered parsing the reason code out of that (we don't want to write it all to Dynamo or it'll cost us 💷 💷 💷) but I'd like to compile some real examples first. For now, this'll give me a prompt to dig into the logs to find out what really happened, and what we could potentially make use of.